EIPS/eip-2035.md

```diff
@@ -0,0 +1,82 @@
+can be introuced without a hard fork). The introduction of the block proofs allows any Ethereum node that wishes to receive them, to process transactions
+in the blocks without needing to access the Ethereum state. All necessary information for the execution (and the proof of validity) is contined in the
+block proofs. In most Ethereum nodes, it will speed up the block processing and reduce the memory footprint of such processing. For mining nodes, however,
+there will be more work to do to construct and transmit the block proofs. Therefore, the extra charge (payable to the miners) is introduced. In the first
+phase, only contract storage will be covered by the block proofs. It means that the Ethereum nodes will still need access to the accounts in the state,
+but block proofs will make it optional to have access to contract storage for executing transactions. Therefore, only `SSTORE` and `SLOAD` opcodes are
+affected.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+There is [empirical analysis](https://github.com/holiman/vmstats/blob/master/README.md) showing that `SLOAD` opcode is currently underpriced in terms
+of execution latency it adds to the block processing. The hypothesis is that it is due to the latency of the database accesses. In the same
+analysis, `SSTORE` is not considered, because its effect on the database accesses can be (and are in many implementations) delayed until the end of
+the block. Stateless clients approach to the contract storage will largely negate that latency because no database accesses will be required.
+Instead, bandwidth consumption goes up. There is emprical analysis (unpublished, but will be) suggesting that 1 uncached `SSTORE` or `SLOAD` adds
+at most 1 kB to the block proofs. At the current cost of data transmission (68 gas per byte), this translates to the increase of gas cost of both
+operations by 69k gas. However, in light of proposal in EIP-2028 (link will be provided when EIP is merged), the increase can be made much smaller.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+Not very formal at the moment, but will be formalised with more research and prototyping. Gas of operations `SLOAD` and `SSTORE` increases by `X` gas
+for those operations that access storage slots that were not previously accessed during the same transaction.
```

Comment:
Can you at least give a list of affected opcodes?

Reply:
The affected opcodes are SLOAD and SSTORE

Reply:
I'm not sure I understand this sentence then. 

> Gas of operations `SLOAD` and `SSTORE` increases by `X` gas for those operations that access storage slots that were not previously accessed during the same transaction.

Is something missing here?

Reply:
You are right, it was not very clear, I changed it now, hope it is better. Thank you!